code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2472,6 +2472,51 @@
 		T.health -= 20
 		T.mutation_mod += 0.1
 
+/datum/reagent/toxin/insecticide
+	name = "Insecticide"
+	id = INSECTICIDE
+	description = "A broad pesticide. Do not ingest!"
+	reagent_state = REAGENT_STATE_LIQUID
+	color = "#49002E" //rgb: 73, 0, 46
+	density = 1.08
+	specheatcap = 4.18
+
+/datum/reagent/toxin/insecticide/reaction_mob(var/mob/living/M, var/method = TOUCH, var/volume)
+
+	if(..())
+		return 1
+	if(iscarbon(M))
+		var/mob/living/carbon/C = M
+		if(!C.wear_mask) //If not wearing a mask
+			C.adjustToxLoss(REM) //4 toxic damage per application, doubled for some reason
```

Comment:
How'd you get 4 toxin per application? REM is 0.5

Reply:
I literally copypasted the code from plantbgone. It does the same level of poisoning since it's based on the same recipe (Toxin+Water=PBG, Toxin+Saltwater=Insecticide)